types/fbinstant/fbinstant-tests.ts

```diff
@@ -0,0 +1,60 @@
+declare const FBInstant: FBInstant;
```

Comment:
Remove this line here, since the tests (or other users of these typings) shouldn't have to add any extra `declare` statements to get the typings to work.

If you change `index.d.ts` to the following, I think things will work as you intended:

```ts
declare namespace FBInstant {
    // put your variables here inside the namespace... 
    let player: Player;
    let context: Context;
    let payments: Payments;

    // ...

    // ... put your functions here inside the same namespace... 
    function getLocale(): string;
    function getPlatform(): string;
    function getSDKVersion(): string;

    // ...

    // .... add your types here in the same namespace ... 
    interface Player {
        getID(): string;
        getSignedPlayerInfoAsync(requestPayload: string): Promise<SignedPlayerInfo>;
   }

   // ...

   // ... add the other types here as well
   type LocalizationsDict = any;
   type SignedPurchaseRequest = string;

   // ...
}
```

Note that I'm using `interface` not class which I think reflects the actual API more accurately.